src/cluster_legacy.h

```diff
@@ -3,6 +3,8 @@
 
 #define CLUSTER_PORT_INCR 10000 /* Cluster port = baseport + PORT_INCR */
 
+#define CLUSTER_DEFAULT_BLACKLIST_TTL 60 /* 1 minute. */
```

Comment:
I doesn't seem like having a default config value defined in a relevant header file is a common practice, so I can remove this if that's preferred

Reply:
It's easier to see the number directly where the config is defined. So, I would say remove this.